.gitignore

```diff
@@ -1,8 +1,21 @@
-# Python Bytecode Cache
+# Binaries
 __pycache__/
+*.py[cod]
 
-# Configuration File
-config.json
+# Dependencies
+src/
 
-# Python Virtual Environment
+# Logging
+*.log
+
+# Secret data
+/config*.yaml
+
+# Deployment
 venv/
```

Comment:
to be fair, we should probably move this to `pipenv`. that would simplify managing the virtual environment, but I think most people on the server haven't used pipenv before. what do you think?